spec/services/catalog/share_info_spec.rb

```diff
@@ -33,17 +34,53 @@
     end
   end
 
-  context "when all group uuids exist" do
-    it_behaves_like "#process"
-  end
+  describe "#process" do
```

Comment:
@eclarizio Do you find it confusing to have the `shared_examples_for` and `describe` block using the same name?  I would probably want to rename the shared_example but not sure what to call it.  Thoughts

Reply:
Yeah I agree. Generally I feel like the outer `describe` should always have the method name, and then a `shared_examples_for` should have some kind of language that flows with the sentence "it behaves like....".

I'll change it, I just kinda glossed over it because I didn't touch anything in it.